src/web/theme/modules/Footer/EnhanceFooter.js

```diff
@@ -1,8 +1,24 @@
 import { graphql } from "react-apollo";
+import compose from "recompose/compose";
+import withProps from "recompose/withProps";
+
+const CATEGORY_ID = 49;
 
 export default FooterQuery =>
-  graphql(FooterQuery, {
-    props: ({ data }) => ({
-      store: data.store
+  compose(
+    withProps(props => ({
+      categoryId: CATEGORY_ID
+    })),
+    graphql(FooterQuery, {
+      options: props => ({
+        variables: {
+          id: props.categoryId
+        }
+      }),
+      props: ({ data }) =>
+        console.log("data", data) || {
+          loading: data.loading,
+          category: data.category
```

Comment:
I would have given a default value to the category, preventing errors in the component if it's null or undefined.